php-practice/section8.php

```diff
@@ -128,3 +139,19 @@ function printGender($gender) {
  *
  * Then create a function that takes a title and echos out a witty comment about that title.
  */
+
+define('Fellowship of the Ring', 1);
+define('Two Towers', 2);
+define('Return of the King', 3);
+
+function printComment($title) {
+	if($title === 'Fellowship of the Ring') {
+		echo 'An incredible start to a trilogy!'.PHP_EOL;
+	} elseif ($title === 'Two Towers') {
+		echo 'Wow! Makes you want to read the Final Book!'.PHP_EOL;
+	} else {
+		echo 'I can\'t believe it\'s over!'.PHP_EOL;
+	}
+}
```

Comment:
So my instructions may have been a little unclear. This doesn't quite do what was intended.

A couple of instructions:

1. Make your constants `UPPER_SNAKE_CASE`:

```php
define('FELLOWSHIP_OF_THE_RING', 1);
```

2. The conditionals inside of the function should be checking against the constants:

```php
if ($title === FELLOWSHIP_OF_THE_RING) {
```